api/database/migrations/2019_10_25_131714_create_forms_field_status_table.php

```diff
@@ -16,6 +16,7 @@ public function up()
         Schema::create('forms_field_status', function (Blueprint $table) {
             $table->bigIncrements('id');
             $table->string('status');
+            $table->timestamps();
```

Comment:
Form field statuses shouldn't have timestamps, please remove. Also we don not edit migrations. [IMPORTANT]